it/test/uk/gov/hmrc/tradergoodsprofilesdatastore/repositories/RecordsRepositorySpec.scala

```diff
@@ -153,6 +275,25 @@ class RecordsRepositorySpec
 
       repository.delete("recordId that does not exist").futureValue mustEqual false
     }
+
+    "when there are inactive records for this eori it must delete them and return num of records deleted" in {
+
+      insert(sampleGoodsItemRecords).futureValue
+      insert(inactiveGoodsItemRecords).futureValue
+
+      val result = repository.deleteInactive(sampleGoodsItemRecords.eori).futureValue
+      result mustEqual 1
+
+      // Making sure record is deleted
```

Comment:
This is fine no need to change but for future note could be a good use of the withClue feature

`withClue("make sure record is deleted"){
            repository.get(inactiveRecordId).futureValue must not be defined
}`

This means if the deletion bit of the test fails the test failure will include "make sure record is deleted" so you know instantly which bit failed